SUMO-289511: updated artifactory doc (#6925)
SumoLogic/sumologic-documentation: docs/integrations/app-development/jfrog-artifactory.md

JFrog Artifactory is a universal artifact repository manager that integrates with CI/CD and DevOps tools to provide artifact tracking. The Sumo Logic app for Artifactory 7 provides insight into your JFrog Artifactory binary repository. Our preconfigured dashboards provide an overview of your system as well as Traffic, Requests and Access, Download Activity, Cache Deployment Activity, and Non-Cached Deployment Activity.

:::info

> This app includes [built-in monitors](#jfrog-artifactory-classic-alerts). For details on creating custom monitors, refer to the [Create monitors for JFrog Artifactory app](#create-monitors-for-jfrog-artifactory-app).

Reviewer: Minor style: house style favors "see" over "refer to," and "the" before a section title reads a little awkward.

```suggestion
This app includes [built-in monitors](#jfrog-artifactory-classic-alerts). For details on creating custom monitors, see [Create monitors for JFrog Artifactory app](#create-monitors-for-jfrog-artifactory-app).
```

Author: Done, Thanks @kimsauce
